puppet/modules/jenkins_job_builder/files/centos.org/pipelines/test/katello/3.8-upgrade.groovy

```diff
@@ -0,0 +1,4 @@
+def playBookVars() {
```

Comment:
Where is this file used?

Reply:
I think it was meant to be added to the job file, but has not yet, both of `3.8-upgrade` and `3.8` have not been added to `foreman-katello-upgrade-3.8-test`.

Reply:
Correct, I missed updating the naming. That has been updated